geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml

```diff
@@ -37,7 +37,7 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-jupyter_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-jupyter-leaflet_${scala.binary.version}</artifactId>
             <scope>runtime</scope>
```

Comment:
Oh, and you are correct in that ``geomesa-jupyter-leaflet`` is just templating; all of its dependencies are things we already have CQs for.